ci-operator/step-registry/hypershift/openstack/aws/nfv/hypershift-openstack-aws-nfv-workflow.yaml

```diff
@@ -0,0 +1,41 @@
+workflow:
+  as: hypershift-openstack-aws-nfv
+  documentation: |-
+    Sets up all the mgmt/infra AWS cluster prerequisites necessary for hypershift openstack guest clusters.
+    Then runs NFV related tests for SR-IOV and DPDK.
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
```

Comment:
Is this autogenerated? Just curious, our project is going away in Jira in favor of just one Control Plane Group project.

Reply:
no it's a copy-paste. maybe we can bulk update it, I see it's used 27 times in the repo.

Reply:
Ok, that can be some follow up later in my opinion.